createdisk.sh

```diff
@@ -108,13 +108,12 @@ ${SSH} core@${VM_IP} 'sudo bash -x -s' <<EOF
 [Unit]
 Description=gvisor-tap-vsock Network Traffic Forwarder
 After=NetworkManager.service
-BindsTo=sys-devices-virtual-net-%i.device
-After=sys-devices-virtual-net-%i.device
 
 [Service]
 Restart=on-failure
 Environment="GV_VSOCK_PORT=1024"
 EnvironmentFile=-/etc/sysconfig/gv-user-network
+ExecStartPre=/bin/sh -c 'for i in {1..10}; do ip link show "\\\$1" && exit 0; sleep 1; done; exit 1' _ %i
```

Comment:
Imo, `After` should achieve this in a better way. You can keep a `ExecStartPre` check which makes sure `%i` is present if you want better error reporting.

The commit log could be updated, with this change, `gv-user-network@.service` will not be restarted when NetworkManager restarts, which explains why the issues we were getting when `gv-user-network@.service` restarts will not be seen.

Reply:
Done, so as per man page of `systemd.netdev` looks like `systemd-network-generator.service is responsible for creating those devices and this make sure the device is available.

Reply:
> > Done, so as per man page of `systemd.netdev` looks like `systemd-network-generator.service is responsible for creating those devices and this make sure the device is available.
> 
> I see nothing related to this in the comments or in the commit:

This is just an info, I wanted to check how all it works :)